packages/functions/src/api/steam.ts

```
@@ -0,0 +1,199 @@
+                                        isLimited: user.isLimitedAccount,
+                                        isVacBanned: user.vacBanned,
+                                        privacyState: user.privacyState as any,
+                                        visibilityState: Number(user.visibilityState),
+                                        tradeBanState: user.tradeBanState.toLowerCase() as any,
+                                    }
+                                })
+
+                            // Does not matter if the user is already there or has just been created, just store the credentials
+                            await Credentials.create({ refreshToken, id: steamID, username })
+
+                            if (!!wasAdded) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)